src/Persistence.php

```diff
@@ -18,6 +18,7 @@ class Persistence
     use \atk4\core\NameTrait;
     use \atk4\core\DiContainerTrait;
 
+    // backward compatibility - will be removed in jul-2021 -->
```

Comment:
Max 1/2 year, we probably want even less (as deprecated code distracts in dev)

I propose to switch to 3 month deprecation period or new minor (d.x.d) ver, so we can deprecate with comments like "will be remove in the next version 2.3"

Reply:
I am still doubtful about the hooks.

Maybe we keep them in Persistence as they were?
Or keep the forwarded like now (do not deprecate)

Reply:
they are related with Sql only, thus deprecating them from Persistence is correct

let's set the depr deriod to 1/2 year

I also mailed Dark to check if we can agree on the "new minor version" depr period for future changes

Reply:
done